### PR TITLE
SAA-1221 overriding Movement type from prison-api as does not adhere to API specification and causing runtime errors.

### PR DESCRIPTION
**Pull request by matt-crowson-moj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/client/prisonapi/overrides/Movement.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+data class Movement(
+  @get:JsonProperty("offenderNo", required = true) val offenderNo: String,
+
+  @get:JsonProperty("createDateTime", required = true) val createDateTime: String,
+
+  @get:JsonProperty("fromAgency", required = true) val fromAgency: String,
+
+  @get:JsonProperty("fromAgencyDescription", required = true) val fromAgencyDescription: String,
+
+  @get:JsonProperty("toAgency", required = false) val toAgency: String?,
```
Comment (matt-crowson-moj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/client/prisonapi/overrides/Movement.kt:19]: The offending field.

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/client/prisonapi/overrides/Movement.kt [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    @JsonProperty("REL")
+    REL("REL"),
+
+    @JsonProperty("TAP")
+    TAP("TAP"),
+
+    @JsonProperty("TRN")
+    TRN("TRN")
+  }
```
Comment (timharrison-moj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/client/prisonapi/overrides/Movement.kt:60]: Is this an exhaustive list of movements?

Comment (matt-crowson-moj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/client/prisonapi/overrides/Movement.kt:60]: It is a direct copy of the original type it overrides.

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**